index.js

```diff
@@ -0,0 +1,7 @@
+const divide = function (a, b) {
+  return a / b;
+}
```

Comment:
You can format this function the same way you did the others:

```suggestion
const divide = (a, b) => a / b;
```

Reply:
Updated the first function to be an arrow function as well as pushed update

Reply:
Cool! For future reference, you're welcome to make changes and push them based on feedback, but you don't NEED to as long as you understand what the comment implies.